src/app/page.tsx

```diff
@@ -3,93 +3,13 @@ import styles from './page.module.css'
 
 export default function Home() {
   return (
-    <main className={styles.main}>
```

Comment:
Why did you remove this code ?

Reply:
I removed it because it was not required